src/routes/JobCreateKBARTRoute.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import compose from 'compose-function';
+import { get } from 'lodash';
+import PropTypes from 'prop-types';
+import { stripesConnect } from '@folio/stripes/core';
+import withFileHandlers from './components/withFileHandlers';
+import View from '../components/views/JobForm';
+
+class JobCreateKBARTRoute extends React.Component {
```

Comment:
This component literally seems to have just one difference, that is in the manifest. So I would say just have 2 different routes pointing to the same component, so

```<Route path={`${path}/create/KBART`} component={JobCreateRoute} />```
    ```<Route path={`${path}/create/JSON`} component={JobCreateRoute} />``` 

and use the `match.path` to extract the format?

That would save a considerable amount of duplication and I think its fairly safe in terms of extensibility as well?

Reply:
I have got this working now, I admit it is probably a lot cleaner, and now I know how the conditional manifest works I can see how easily extensible this will be, so no problems here :)